docs/eventing/broker/README.md

```diff
@@ -36,6 +36,50 @@ metadata:
 EOF
 ```
 
+## Explicit Broker configuration
+
+You can of course create a Broker without relying on the defaults (as described
+above) by specifying the `BrokerClass` by using an annotation to specify which
+Broker implementation you'd like to use:
+
+```yaml
+kind: Broker
+metadata:
+  annotations:
+    eventing.knative.dev/broker.class: MTChannelBasedBroker
+```
+
+and explicitly configuring the spec.config, for example:
+```yaml
+kind: Broker
+spec:
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: config-br-default-channel
+    namespace: knative-eventing
+```
+
+A full example could look something like this:
+
+```shell
+kubectl create -f - <<EOF
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  annotations:
+    eventing.knative.dev/broker.class: MTChannelBasedBroker
+  name: default
+  namespace: default
+spec:
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: config-br-default-channel
+    namespace: knative-eventing
+EOF
+```
```

Comment:
Move this to the intro part maybe?

Reply:
Ok, so we have the two steps you can modify, class / config. Then showing them combined? How's that?

Reply:
I think it's actually fine as is now I see it with the other changes :slightly_smiling_face: thank you!

Reply:
One last thing - I would actually remove the `kubectl create -f - <<EOF` bits so that it's just showing the YAML example. That's up to you though :slightly_smiling_face:

Reply:
ok, done :)